### PR TITLE
Add biscuit rules

### PR DESCRIPTION
**Pull request by QuentinBisson**
I generated the rules using the transform tool and tested them all against the old and new prometheus setups

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  - `ControlPlaneSystemdUnitFailed`
+  - `ControlPlaneCriticalSystemdUnitFailed`
+  - `ControlPlaneDisabledSystemdUnitActive`
+  - `ControlPlaneHighNumberSystemdUnits`
+  - `EtcdDown`
+  - `GatekeeperDown`
+  - `NoHealthyJumphost`
+  - `NodeVaultTokenAlmostExpired`
+  - `VaultIsDown`
+  - `VaultIsSealed`
```
Comment (TheoBrigitte) [inline CHANGELOG.md:39]: Alphabetical ordering make sense here.

File: helm/prometheus-meta-operator/templates/prometheus-rules/crsync.rules.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        opsrecipe: crsync-too-many-tags-missing/
+      expr: crsync_sync_tags_total{registry="quay.io"} - on (repository,app) group_left sum by(repository,app) (crsync_sync_tags_total{registry!="quay.io"}) > 0
+      for: 1h
+      labels:
+        area: kaas
+        cancel_if_outside_working_hours: "true"
+        severity: page
+        team: biscuit
+        topic: releng
+{{- end }}
```
Comment (TheoBrigitte) [inline helm/prometheus-meta-operator/templates/prometheus-rules/crsync.rules.yml:41]: Why no new line at end of file ?

File: helm/prometheus-meta-operator/templates/prometheus-rules/deployment.rules.yml [hunk 3]
```diff
@@ -38,3 +39,29 @@ spec:
...
+        description: '{{`Deployment {{ $labels.namespace}}/{{ $labels.deployment }} is not satisfied.`}}'
+        opsrecipe: deployment-not-satisfied-china/
+      expr: kube_deployment_status_replicas_unavailable{cluster_type="control_plane", deployment!~"coredns|nginx-ingress-controller|alertmanager.*|prometheus.*|prometheus-meta-operator.*|promxy.*|.+-admission-controller-unique", cluster_id=~"axolotl|giraffe|argali"} > 0
+      for: 3h
+      labels:
+        area: kaas
+        severity: page
+        team: biscuit
+        topic: controlplane
+
```
Comment (TheoBrigitte) [inline helm/prometheus-meta-operator/templates/prometheus-rules/deployment.rules.yml:67]: Why extra new line at end of file ?

**Review by TheoBrigitte — APPROVED** [review]
Fine with me since this was auto-generated

**Comment by QuentinBisson** [comment]
I addressed your review but we are waiting for cortex to be working first :)

**Comment by QuentinBisson** [comment]
@giantswarm/team-biscuit it looks like we can add all rules apart from NoHealthyJumphost because the new monitoring setup is not using the host network (cf. https://gigantic.slack.com/archives/C3S482Z98/p1611916110021000)

Can we go ahead here and merge this but without the NoHealthyJumphost alert and keep it in g8s-prometheus?

I will create an issue to enable scrape to the bastion machine for every provider on your board as a follow up

**Comment by kopiczko** [comment]
If there is a question there is a problem. Why shouldn't we want it?

**Comment by QuentinBisson** [comment]
Well, we all want it, but as it is, we cannot merge because the new prometheus do not have access to the  bastion hosts. Unlocking the situation will take time as this needs a roll of all CPs so would it be possible to merge all of them without the alert for now and when the issue is gone, we can migrate the NoHealthyJumphost alert then. This alert will stay in g8s-prometheus for now though so you will still receive alerts

**Pull request merged**